osschallenge/tests/test_selenium_not_logged_in.py

```diff
@@ -114,7 +114,3 @@ def test_register_passwords_do_not_match(self):
     def test_redirect_rankup(self):
         self.rankup_page.open()
         self.driver.find_element_by_id('id_username')
-
-    def test_no_redirect_profile(self):
```

Comment:
I think this is actually a "valid" test.. if you rank up, then you will not be on the profile page and this is checked with the `find_element_by_id('facebook')`

Reply:
I have changed the tests.